### DISCUSSION TITLE
After the update, chia unable to synchronize nodes.

### DISCUSSION THREAD

**Discussion by caroline319** [Chia-Network/chia-blockchain#17175] [Support]
After updating version 2.1.3, Chia unable to synchronize nodes. The computer's operating system is Windows Server 2019. How to  solve i

**Comment by BrandtH22** [comment]
Hey @caroline319 , as a heads up the best place to reach our support team is on Discord (https://discord.gg/chia). 

We will need a bit more information regarding the sync issues you are experiencing to better assist:
Are you seeing any errors in the logs? 
Have you tried the new release candidate version? (https://github.com/Chia-Network/chia-blockchain/releases/tag/2.1.4-rc1)
Have you tried creating a new config file?

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
